src/test/java/org/jenkinsci/plugins/workflow/cps/actions/ArgumentsActionImplTest.java

```diff
@@ -156,18 +156,20 @@ public void testStringSafetyTest() throws Exception {
         String input = "I have a secret p4ssw0rd";
         HashMap<String,String> passwordBinding = new HashMap<>();
         passwordBinding.put("mypass", "p4ssw0rd");
-        Assert.assertTrue("Input with no variables is safe", ArgumentsActionImpl.isStringSafe(input, new EnvVars(), Collections.EMPTY_SET));
-        Assert.assertFalse("Input containing bound value is unsafe", ArgumentsActionImpl.isStringSafe(input, new EnvVars(passwordBinding), Collections.EMPTY_SET));
-
-        Assert.assertTrue("EnvVars that do not occur are safe", ArgumentsActionImpl.isStringSafe("I have no passwords", new EnvVars(passwordBinding), Collections.EMPTY_SET));
-
-        HashMap<String, String> safeBinding = new HashMap<>();
-        safeBinding.put("harmless", "secret");
-        HashSet<String> safeVars = new HashSet<>();
-        safeVars.add("harmless");
-        passwordBinding.put("harmless", "secret");
-        Assert.assertTrue("Input containing whitelisted bound value is safe", ArgumentsActionImpl.isStringSafe(input, new EnvVars(safeBinding), safeVars));
-        Assert.assertFalse("Input containing one safe and one unsafe bound value is unsafe", ArgumentsActionImpl.isStringSafe(input, new EnvVars(passwordBinding), safeVars));
+        Set<String> sensitiveVariables = new HashSet<>();
+        sensitiveVariables.add("mypass");
+        Assert.assertFalse("Input with no variables is safe", ArgumentsActionImpl.isStringSensitive(input, new EnvVars(), sensitiveVariables));
+        Assert.assertTrue("Input containing bound value is unsafe", ArgumentsActionImpl.isStringSensitive(input, new EnvVars(passwordBinding), sensitiveVariables));
+
+        Assert.assertFalse("EnvVars that do not occur are safe", ArgumentsActionImpl.isStringSensitive("I have no passwords", new EnvVars(passwordBinding), sensitiveVariables));
+
+//        HashMap<String, String> safeBinding = new HashMap<>();
+//        safeBinding.put("harmless", "secret");
+//        HashSet<String> safeVars = new HashSet<>();
+//        safeVars.add("harmless");
+//        passwordBinding.put("harmless", "secret");
+//        Assert.assertTrue("Input containing whitelisted bound value is safe", ArgumentsActionImpl.isStringSafe(input, new EnvVars(safeBinding), safeVars));
+//        Assert.assertFalse("Input containing one safe and one unsafe bound value is unsafe", ArgumentsActionImpl.isStringSafe(input, new EnvVars(passwordBinding), safeVars));
```

Comment:
If this is obsolete, just delete it, do not comment it out.

Reply:
yeah, i meant to, just forgot to do that in the clean up.